cmd/crank/main.go

```diff
@@ -65,6 +65,14 @@ var cli struct {
 	Push    pushCmd    `cmd:"" help:"Push Crossplane packages."`
 
 	XPKG xpkg.Cmd `cmd:"" help:"Crossplane package management."`
+
+	Alpha struct {
+		// Add here alpha subcommands
+	} `cmd:"" help:"Alpha features. WARN: May be changed or removed without notice"`
+
+	Beta struct {
+		// Add here beta subcommands
+	} `cmd:"" help:"Beta features. WARN: May be changed or deprecated in a future release"`
```

Comment:
Should we put the function related commands under here? I don't feel strongly, but it was where I intuitively expected them to be since Functions overall is a beta feature.

Reply:
Yes, I was thinking about that too https://github.com/crossplane/crossplane/pull/4736#issuecomment-1752881163, I'll update the other PRs when this is merged